src/Models/Round.php

```diff
@@ -130,6 +131,11 @@ public function hasEnded()
         return ($this->end_date <= now());
     }
 
+    public function hasOffensiveActionsDisabled()
```

Comment:
```suggestion
    /**
     * Returns whether a round has offensive actions disabled.
     *
     * Actions are disabled near the end of the round to prevent suicides and whatnot.
     */
    public function hasOffensiveActionsDisabled(): bool
```

Explicit return type and docblock